presto-docs/src/main/sphinx/connector/iceberg.rst

```diff
@@ -247,6 +247,8 @@ Property Name                                           Description
                                                         metadata optimization is skipped.
 
                                                         Set to ``0`` to disable metadata optimization.
+
+``iceberg.split-manager-threads``                       Number of threads to use for generating iceberg splits.        ``Number of available processors``
```

Comment:
```suggestion
``iceberg.split-manager-threads``                       Number of threads to use for generating Iceberg splits.        ``Number of available processors``
```

Reply:
Nice catch @steveburnett, I've already fixed it.